kotlinlib/src/mill/kotlinlib/KotlinModule.scala

```diff
@@ -102,19 +102,48 @@ trait KotlinModule extends JavaModule { outer =>
     )().toSeq ++ kotlinWorkerClasspath()
   }
 
+  /** Flag to use the embeddable kotlin compiler */
```

Comment:
```suggestion
  /** 
   * Flag to use the embeddable kotlin compiler. 
   * This can be necessary to avoid classpath comflicts or ensure 
   * compatibility to the used set of plugins. 
   *
   * See also https://discuss.kotlinlang.org/t/kotlin-compiler-embeddable-vs-kotlin-compiler/3196
   */
```